andrey_n/homework_5/homework_5/main.py

```diff
@@ -0,0 +1,148 @@
+def double_input(input_data):
+    """Doubles input"""
+    return input_data * 2
+
+
+def clean(item: str, markers: list) -> str:
+    """Delete comments from item"""
+    for mark in markers:
+        if mark in item:
+            return item[: item.index(mark)]
```

Comment:
what will be returned by `index` if there is no mark in the item?

Reply:
If item without marks it's return item. Line11